pkgs/by-name/im/imhex/package.nix

```diff
@@ -73,31 +86,48 @@ stdenv.mkDerivation rec {
     mbedtls
     nlohmann_json
     yara
+  ] ++ lib.optionals stdenv.isDarwin [
+    darwin.apple_sdk_11_0.frameworks.UniformTypeIdentifiers
   ];
 
   # autoPatchelfHook only searches for *.so and *.so.*, and won't find *.hexpluglib
   # however, we will append to RUNPATH ourselves
-  autoPatchelfIgnoreMissingDeps = [ "*.hexpluglib" ];
-  appendRunpaths = [
+  autoPatchelfIgnoreMissingDeps = lib.lists.optional stdenv.isLinux [ "*.hexpluglib" ];
+  appendRunpaths = lib.lists.optional stdenv.isLinux [
```

Comment:
I haven't come across `lib.ists.optional` in my nixpkgs work so far and will need to read up on it. Mind providing some context as to why this was used here instead of just `lib.optionals`, @pcasaretto? 🙂

Reply:
It was incorrect. Thanks for catching that!`optional` would have inserted the list inside another list.
However, I fail to understand how `optional` and `optionals` are accessible in the root `lib`.
From the documentation they seem to belong to the `lists`  subattribute.

Reply:
A lot of the functions are aliased into lib. You can use noogle.dev to check if a function is aliased, for example see the bottom list of aliases for optional:

https://noogle.dev/f/lib/lists/optional

Reply:
Bookmarked!